File: Beecrowd/1101 Sequência de Números e Soma/main.cpp
#include <iostream>
 
using namespace std;

int main(){
    int M, N, aux, i, soma;
    
    while(scanf("%d %d", &M, &N) !=EOF){
        soma = 0;
        if(M <= 0 || N <= 0){
            break;
        }
        if(M > N){
            aux = M;
            M = N;
            N = aux;
        }
    
        for(i = M; i <= N; i++){
            soma += i;
            cout << i << " ";
        }
            cout << "Sum=" << soma << endl;
    }

    return 0;
}

File: Beecrowd/1101 Sequência de Números e Soma/main.py
while(True):
    entrada = [int(a) for a in input().split()]
    M = entrada[0]
    N = entrada[1]
    soma = 0
        
    if M <= 0 or N <= 0:
        break

    if M > N:
        aux = M
        M = N
        N = aux

    for i in range(M, N + 1):
        soma += i
        print(i, end = ' ')
    print("Sum={}".format(soma))